roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -187,12 +188,29 @@ def run_check(name, check, user_disabled_checks):
     if not is_active:
         return dict(skipped=True, skipped_reason="Not active for this host")
 
+    output_dir = check.get_var("openshift_checks_output_dir", default=None)
```

Comment:
I thought for a while about how to regulate whether the files are created and where they go. I decided against env vars because using them seems pretty clunky and using inventory variables allows variation per host.